src/View/View.php

```diff
@@ -110,14 +108,6 @@ class View implements EventDispatcherInterface
      */
     public $name;
 
-    /**
-     * Current passed params. Passed to View from the creating Controller for convenience.
-     *
-     * @var array
-     * @deprecated 3.1.0 Use `$this->request->getParam('pass')` instead.
-     */
-    public $passedArgs = [];
```

Comment:
Should also remove the code that sets this property.

Reply:
@markstory , can you clarify a bit? Where would that code be?

Reply:
@iliepandia  all usage of `passedArgs` in view 
https://github.com/cakephp/cakephp/blob/843c5a480824c5429b13dd39c7437afb0e01072d/src/View/View.php#L236

and in tests

Reply:
I have removed the reference from the line 236. There are indeed more references to this and tests, but they related to how the `Controller` builds the view and I did not touch on that. I assumed that each package fix will only work on their respective package files so I did not change how the controller works.